tests/api/test_product_minimal_variant_price.py

```diff
@@ -600,6 +603,8 @@ def test_sale_delete_updates_products_minimal_variant_prices(
         }
     }
     """
+    sale.products.add(product)
+    sale.save()
```

Comment:
Are you actually editing the sale? Seems like you only act on a M2M

Reply:
sorry my bad